.ci/create-backstop-json-from-default.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
```

Comment:
It would be great to have some comments in this file explaining when it is called (and from where) and what it is for.

Reply:
Rather than comments that simply say what the next line is doing, I had in mind something more like:

> This file is called by visual-regression-test.sh whenever the user has not already provided
> a Backstop configuration file.  In this instance, a default configuration file will be written from
> a template.